kube_ops_view/main.py

```diff
@@ -45,7 +45,8 @@
     request_token_url=None,
     access_token_method='POST',
     access_token_url=os.getenv('ACCESS_TOKEN_URL'),
-    authorize_url=AUTHORIZE_URL
+    authorize_url=AUTHORIZE_URL,
+    request_token_params={'scope': os.getenv('SCOPE')}
```

Comment:
Should we make the `request_token_params` option if no scope is given? That would make the change fully backwards-compatible (there might be a difference between sending no params and sending an empty `scope` param).

Reply:
Good idea, probably there might be issues with some vendors if we pass scope as None in request_token_params. I have committed changes to make the request_token_params option None if no scope is given. 
ee256bca7292a0039140579ed48883ecf1829529